libs/react-helpers/src/lib/format/number.ts

```diff
@@ -0,0 +1,27 @@
+import { BigNumber } from 'bignumber.js';
```

Comment:
follow up: do we want to make a utils/helpers package that is just TS rather than making this specific to react?

Reply:
Yeah IMO I think we can just rename react-helpers to utils?

Reply:
Lets do this as follow up 👍